api/room/models.py

```diff
@@ -39,6 +39,7 @@ class Room(Base, Utility):
     firebase_token = Column(String, nullable=True)
     cancellation_duration = Column(Integer, default=10)
     state = Column(Enum(StateType), default="active")
+    next_sync_token = Column(String, nullable=True)
```

Comment:
The `next_sync_token` enables Incremental sync to be performed repeatedly and updates the client with all the changes that happened ever since the previous sync.